src/Restore.php

```diff
@@ -729,4 +729,31 @@ private function restoreTableColumnsMetadata(array $tableInfo, string $tableId,
             $metadataClient->postTableMetadataWithColumns($tableMetadataUpdateOptions);
         }
     }
+
+    public function restorePermanentFiles(): void
+    {
+        $this->logger->info('Downloading permanent files');
+        $fileContent = $this->getDataFromStorage('permanentFiles.json');
+
+        $permanentFiles = json_decode((string) $fileContent, true);
+
+        $tmp = new Temp();
+        $tmp->initRunFolder();
+
```

Comment:
@ondrajodas pls, mohli by sme v ramci dryRun vypisat aj ich pocet?

Reply:
Or we can have information above "Downloading %count permanent files"

Reply:
doplněno